app/views/smart_answers/_previous_answers.html.erb

```diff
@@ -2,7 +2,12 @@
   <table class="govuk-table previous-answers-table">
     <caption class="govuk-table__caption">
       <h2 class="govuk-heading-m">Previous answers</h2>
-      <p class="govuk-body"><%= link_to "Start again", restart_flow_path(@presenter), :class => "govuk-link" %></p>
+      <p class="govuk-body"><%= link_to "Start again", restart_flow_path(@presenter), :class => "govuk-link", :data => {
+        module: "track-click",
+        "track-action": "Start again",
+        "track-category": "StartAgain",
+        "track-label": @presenter.current_node.title
+      } %></p>
```

Comment:
Very minor, but I think statements should either be multi-line (with each clause on a new line) or single line and not a combination of the two as here. I'd suggest this would be a little clearer:
```
<p class="govuk-body">
  <%= link_to "Start again", 
    restart_flow_path(@presenter), 
    class: "govuk-link", 
    data: {
      module: "track-click",
      "track-action": "Start again",
      "track-category": "StartAgain",
      "track-label": @presenter.current_node.title
    } 
  %>
</p>